os/fs/smartfs/smartfs_smart.c

```diff
@@ -1490,6 +1490,11 @@ static int smartfs_bind(FAR struct inode *blkdriver, const void *data, void **ha
 	}
 
 	*handle = (void *)fs;
+	
+	ret = smartfs_sector_recovery(fs);
+	if (ret != 0) {
```

Comment:
`smartfs_sector_recovery` returns `OK` or errno. How about comparing with `OK` instead of `0`?

Reply:
Yes you're right. I already know that, some of in this file use '0' instead of OK, I just aligned with above line's code style(line 1488).
it will be cleaned totally, after https://github.com/Samsung/TizenRT/pull/4376 merged, may in someday =D